devops/scripts/install_drivers.sh

```diff
@@ -114,20 +114,16 @@ InstallIGFX () {
     | grep ".*deb" \
     | wget -qi -
   get_release intel/compute-runtime $CR_TAG \
-    | grep -E ".*((deb)|(sum))" \
+    | grep -E ".*((\.deb)|(sum))" \
     | wget -qi -
-  # Perform the checksum conditionally and then get the release
-  # Skip the ww45 checksum because the igc_dev driver was manually updated
-  # so the package versions don't exactly match.
-  if [ ! -f "ww45.sum" ]; then
-      sha256sum -c *.sum
-  fi
+  # We don't download .ddeb packages, so ignore missing ones.
+  sha256sum -c *.sum --ignore-missing
   get_release intel/cm-compiler $CM_TAG \
     | grep ".*deb" \
     | grep -v "u18" \
     | wget -qi -
   get_release oneapi-src/level-zero $L0_TAG \
-    | grep ".*$UBUNTU_VER.*deb" \
+    | grep ".*$UBUNTU_VER.*deb$" \
```

Comment:
do we need to add the ddeb check here too?

Reply:
No, actually compute runtime is the only dependency containing debug packages.